nessie/sql_templates/create_sis_advising_notes_schema.template.sql

```diff
@@ -308,7 +308,8 @@ AS (
     ON N.note_category = C.note_category
     JOIN
         {redshift_schema_sis_advising_notes}.advising_note_subcategories S
-    ON N.note_subcategory = S.note_subcategory
+    ON N.note_category = S.note_category
+    AND N.note_subcategory = S.note_subcategory
```

Comment:
@lyttam I still see duplicates w/ the `subcategory` condition above. What about a DISTINCT constraint?